src/tests/Interop/PInvoke/Decimal/DecimalTest.csproj

```diff
@@ -7,5 +7,8 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="CMakeLists.txt" />
+    <ProjectReference Include="$(RepoRoot)/src/tests/Common/XUnitWrapperGenerator/XUnitWrapperGenerator.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
What is your thinking about this bit? Do you expect this to stay in the test projects being converted or is this just a temporary shortcut for your proof of concept we'll clean up somehow? I was originally hoping that we would just make a property adjustment to tag the tests as conforming to the new model (e.g. by marking them as <code>BuildAndRun / Library</code> that I believe to be invalid today) where the msbuild scripts initially do the additional generated wrapper magic for such tests and ultimately we'll compile these tests in a completely separate phase to facilitate compiling multiple tests into a single app in accordance with JanK's suggestion.